### PR TITLE
[MIG] web_sheet_full_width: migration to 11.0

### PR DESCRIPTION
**Pull request by Jerther**
Since the chatter is now on the right side of the screen, the form cannot take the whole screen as it used to, but when there is no chatter or when it is small, this addon is still useful.

I could try and make the chatter sizable with the mouse if that's still within the scope of this module. The chatter width could then be saved and used globally. Thoughts welcome :)

### CODE REVIEW

**Comment by pedrobaeza** [comment]
For smallers screens where the chatter is not at the right side, it still should be full screen.

**Comment by Jerther** [comment]
Oh, responsive design! Nice!

**Comment by mgielissen** [comment]
@Jerther Chatter on the right screen is only in EE

**Comment by Jerther** [comment]
And it needed the extra CSS :) Thanks @mgielissen

**Review by oihane — APPROVED** [review]
LGTM

**Comment by yajo** [comment]
I'd be glad to see this integrated this into `web_responsive` and delete this addon.

**Comment by pedrobaeza** [comment]
No, because there's people work enterprise interface that also wants it.

File: web_sheet_full_width/README.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+to get full width in the form view sheet.
+
+This module works in community edition and in enterprise edition.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/162/10.0
```
Comment (yajo) [inline web_sheet_full_width/README.rst:19]: Wrong version

File: web_sheet_full_width/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "version": "11.0.1.0.1",
+    "author": "Therp BV, Sudokeys, GRAP, Métal Sartigan, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "summary": "Use the whole available screen width when displaying sheets",
+    "category": "Tools",
+    "depends": [
+        'web',
+    ],
+    "data": [
+        "view/qweb.xml",
```
Comment (yajo) [inline web_sheet_full_width/__manifest__.py:14]: I think this should be `templates/assets.xml` instead.

**Review by yajo — CHANGES_REQUESTED** [review]
Missing commit history. Check out https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

File: web_sheet_full_width/__manifest__.py (hunk 2 above)
Comment (Jerther) [reply web_sheet_full_width/__manifest__.py:14]: I thought it was weird... I'll sure rename this.

File: web_sheet_full_width/README.rst (hunk 1 above)
Comment (Jerther) [reply web_sheet_full_width/README.rst:19]: That of web_responsive is also wrong

Comment (pedrobaeza) [reply web_sheet_full_width/README.rst:19]: Indeed, changed in https://github.com/OCA/web/commit/ebdbd58e2b7ff0dc6285e4a0fa301387fa674609

**Pull request closed without merging**